src/continuum_opacity/opacity_metal.jl

```diff
@@ -0,0 +1,561 @@
+# the only reason that we NEED to load in the electron_configurations data is that it's not
+# completely clear to me exactly how I would compute certain quantities (i.e. the statistical weight and the ground state energy)
+
+include("../Korg.jl")
+using Base
+using Interpolations
+# import Korg.partition_funcs, Korg.atomic_symbols, Korg.numerals, and constants
+
+
+# While the StateID struct and its helper functions may seem a little over-engineered, they
+# primarily exist to document the iSLP and iLV quantities used by TOPBase
+
+"""
+    StateID(iSLP, iLV)
+
+`StateID` represents a given state (electron configuration) of an ion species. The arguments are
+taken from the TOPBase.
+
+# Arguments
+- `iSLP::UInt16`: An integer representing the quantum atomic numbers SLπ (total spin, orbital
+  angular momentum, and parity). The value is given by `100*(2*S+1) + 10*L + π`. Note that TOPBase
```

Comment:
Forgive my ignorance, why not `100*S`?

Reply:
I'm speculating, but maybe S can be equal to 0?

Reply:
In any case, this data-structure is reflecting the format used within the Opacity Project (I'm not really planning to change the format...)

Reply:
I just confirmed that the ground state of neutral Helium has S = 0